File: Fourth Semester/Systems-for-Design-and-Implementation/Lab04/Lab04-Java/src/main/java/domain/Participant.java
package domain;

public class Participant implements Identifiable<Integer>{
    private String nume;
    private Integer varsta;


    public Participant(String nume, Integer varsta) {
        this.nume = nume;
        this.varsta = varsta;

    }

    public String getNume() {
        return nume;
    }

    public Integer getVarsta() {
        return varsta;
    }


    public void setNume(String nume) {
        this.nume = nume;
    }

    @Override
    public void setId(Integer integer) {

    }

    @Override
    public Integer getId() {
        return null;
    }
}

File: Fourth Semester/Systems-for-Design-and-Implementation/Lab04/Lab04-C#/Lab04/domain/Participant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab03.domain
{
    public class Participant : Identifiable<int>
    {

        private String nume;

        private int varsta;

        public Participant(String nume, int varsta)
        {
            this.nume = this.nume;
            this.varsta = this.varsta;
            
        }

        public String getNume()
        {
            return this.nume;
        }

        public int getVarsta()
        {
            return this.varsta;
        }
        

        public void setNume(String nume)
        {
            this.nume = this.nume;
        }

      

        public void setId(int id)
        {
            throw new NotImplementedException();
        }

        int Identifiable<int>.getId()
        {
            throw new NotImplementedException();
        }
    }
   
}
